src/include/uapi/ipc/topology.h

```diff
@@ -64,6 +64,7 @@ enum sof_comp_type {
 	SOF_COMP_EQ_FIR,
 	SOF_COMP_FILEREAD,	/**< host test based file IO */
 	SOF_COMP_FILEWRITE,	/**< host test based file IO */
+	SOF_COMP_KPB,
```

Comment:
Add a comment here to explain what KPB means. First readers won't know that.

Reply:
Ok.